src/Playwright/Transport/Channels/Root.cs

```diff
@@ -39,9 +40,9 @@ internal Root(IChannelOwner parent, Connection connection, string guid) : base(p
 
     internal async Task<PlaywrightImpl> InitializeAsync()
     {
-        var args = new
+        var args = new Dictionary<string, object>
```

Comment:
`<string, string>`?

Reply:
Parameters are always `<string, object>` ensured on the type level.